packages/flutter/lib/src/material/progress_indicator.dart

```diff
@@ -417,15 +417,18 @@ class LinearProgressIndicator extends ProgressIndicator {
   /// When true, the [LinearProgressIndicator] will use the 2023 Material 3
   /// Design appearance.
   ///
-  /// Defaults to true. If false, the [LinearProgressIndicator] will use the
+  /// If null, then the [ProgressIndicatorThemeData.year2023] will be used.
+  /// If that is null, then defaults to true.
+  ///
+  /// If this is set to false, the [LinearProgressIndicator] will use the
   /// latest Material 3 Design appearance, which was introduced in December 2023.
```

Comment:
nit: I feel like it's either called Material Design 3 or Material 3:)

Reply:
Great catch!